backend/drm/drm.c

```diff
@@ -1004,6 +1004,21 @@ static void drm_connector_get_cursor_size(struct wlr_output *output,
 	*height = (int)drm->cursor_height;
 }
 
+static const struct wlr_drm_format_set *drm_connector_get_primary_formats(
+		struct wlr_output *output, uint32_t buffer_caps) {
+	if (!(buffer_caps & WLR_BUFFER_CAP_DMABUF)) {
+		return NULL;
+	}
+	struct wlr_drm_connector *conn = get_drm_connector_from_output(output);
+	if (!conn->crtc) {
+		return false;
```

Comment:
Why not return `NULL`?

Reply:
Good catch, seems like it's copypasta coming from `get_cursor_formats`. Sending a separate PR to fix these.